CHANGELOG.md

```diff
@@ -1,20 +1,17 @@
-# NEXT RELEASE
+# 10.0.0-beta.7 Release notes
 
 ### Enhancements
 * Added ability to replace last sort descriptor on DescriptorOrdering in addition to append/prepending to it [#3884]
 * Add Decimal128 subtraction and multiplication arithmetic operators.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
- 
-### Breaking changes
-* None.
+* Issues fixed by releases v6.0.24 to v6.0.25:
+  * If you have a realm file growing towards 2Gb and have a table with more than 16 columns, then you may get a "Key not found" exception when updating an object. If asserts are enabled at the binding level, you may get an "assert(m_has_refs)" instead. ([#3194](https://github.com/realm/realm-js/issues/3194), since v6.0.0)
```

Comment:
Then we also need to update the "Since xx" version number to reflect where in v10 the problem was seen first.

Reply:
It is still "since v6.0.0"